admin/admin-model.php

```diff
@@ -282,6 +282,40 @@ public function update_language( $args ) {
 		return true;
 	}
 
+	/**
+	 * Update language description where some of language properties are stored.
+	 *
+	 * @param array $args {
+	 *   @type int    $lang_id        Id of the language to modify.
+	 *   @type string $name           Language name ( used only for display ).
+	 *   @type string $slug           Language code ( ideally 2-letters ISO 639-1 language code ).
+	 *   @type string $locale         WordPress locale. If something wrong is used for the locale, the .mo files will not be loaded...
+	 *   @type int    $rtl            1 if rtl language, 0 otherwise.
+	 *   @type int    $term_group     Language order when displayed.
+	 *   @type string $flag           Optional, country code, @see flags.php.
+	 * }
+	 * @return mixed The serialized description array updated.
+	 */
+	protected function update_language_description( $args ) {
+		$language_term = get_term( (int) $args['lang_id'] );
+
+		$old_description = array();
+
+		if ( $language_term instanceof WP_Term ) {
+			$description = maybe_unserialize( $language_term->description );
+			if ( is_array( $description ) ) {
+				$old_description = $description;
+			}
+		}
```

Comment:
Proposing a simplified way to avoid an intermediate variable:
```suggestion
		if ( $language_term instanceof WP_Term ) {
			$old_description = maybe_unserialize( $language_term->description );
		}

		if ( empty( $old_description ) || ! is_array( $old_description ) ) {
				$old_description = array();
		}
```

Reply:
Done in 7095e98d027f964cd25d2007152f0b6589234335